<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSttBinRecordCbWidget</class>
 <widget class="QWidget" name="QSttBinRecordCbWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>558</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>QSttBinRecordCbWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="m_HLayOut_Funs"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="m_HLayOut_Btns">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>选择插件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_Label_SelFun">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>选择功能</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QRadioButton" name="m_RBtn_ShowPrim">
       <property name="text">
        <string>显示一次值</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_Btn_Back">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
